search-solr/src/main/java/edu/unc/lib/boxc/search/solr/services/MultiSelectFacetListService.java

```diff
@@ -119,6 +121,28 @@ public SearchResultResponse getFacetListResult(SearchRequest searchRequest) {
         return resultResponse;
     }
 
+    /**
+     * Finds and returns the minimum year created for the given query
+     * @param selectedState
+     * @param originalRequest
+     * @return String
+     */
+    public String getMinimumDateCreatedYear(SearchState selectedState, SearchRequest originalRequest) {
+        selectedState.setRowsPerPage(1);
+        selectedState.setResultFields(Collections.singletonList(SearchFieldKey.DATE_CREATED_YEAR.name()));
+        selectedState.setSortNormalOrder(false);
```

Comment:
You still need to specify the sort type in addition to reversing the order, so add in
```
selectedState.setSortType("dateCreated");
```